### PR TITLE
GH-21

### PR DESCRIPTION
**Pull request by Skater901**
Added parsing for Javax/Jakarta validation annotations to populate the Swagger schema with requirements for the API.

I created a new library for it as that seemed to be the best approach (the existing `schema-kenerator-swagger` seems to be purely for parsing Swagger annotations, and there's the `schema-kenerator-jackson-swagger` library), and also because to cover both annotations, I had to bring in both the `jakarta.validation:jakarta.validation-api`, and bump the version of `javax.validation:validation-api`. Keeping that separate from all the other libraries seems best.

I don't know if the README and `build.gradle.kts` file are correct; I copied them from other libraries and made what I think are the correct adjustments, so please review those carefully.

### CODE REVIEW

File: schema-kenerator-validation-swagger/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Schema-Kenerator Validation Swagger
+
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.github.smiley4/schema-kenerator-validation-swagger/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.github.smiley4/schema-kenerator-validation-swagger)
```
Comment (Skater901) [inline schema-kenerator-validation-swagger/README.md:3]: I don't know how this `badge.svg` file actually gets created... I'm assuming it's automatic, but if not, let me know what I need to change to make this bit work.

File: schema-kenerator-validation-swagger/build.gradle.kts
Comment (Skater901) [inline schema-kenerator-validation-swagger/build.gradle.kts:1]: I copied most of this from the `schema-kenerator-core/build.gradle.kts` and `schema-kenerator-swagger/build.gradle.kts` files.

File: schema-kenerator-validation-swagger/README.md (hunk 1 above)
Comment (SMILEY4) [reply schema-kenerator-validation-swagger/README.md:3]: tbh, i'm not 100% sure either right now, i think it gets created with the maven central release - we'll see with the release :)

**Comment by SMILEY4** [comment]
Thanks for the pr - looks all good to me. :+1:

**Pull request merged**